cljfmt/src/cljfmt/core.cljc

```diff
@@ -213,17 +234,24 @@
   (and (> depth 0)
        (= (inc idx) (index-of (nth (iterate z/up zloc) depth)))))
 
-(defn- fully-qualify-symbol [possible-sym alias-map]
-  (if-let [ns-string (and (symbol? possible-sym)
-                          (namespace possible-sym))]
-    (symbol (get alias-map ns-string ns-string)
-            (name possible-sym))
-    possible-sym))
+(defn- fully-qualified-symbol [zloc alias-map]
```

Comment:
What about writing it instead as:

```clojure
(defn- qualify-symbol-by-alias-map [possible-sym alias-map]
  (when-let [ns-string (namespace possible-sym)]
    (symbol (get alias-map ns-string ns-string)
            (name possible-sym))))

(defn- qualify-symbol-by-ns-form [possible-sym zloc]
  (when-let [ns-name (find-namespace zloc)]
    (symbol (name ns-name) (name possible-sym))))

(defn- fully-qualified-symbol [zloc alias-map]
  (let [possible-sym (form-symbol zloc)]
    (or (if-not (symbol? possible-sym) possible-sym)
        (qualify-symbol-by-alias-map possible-sym alias-map)
        (qualify-symbol-by-ns-form possible-sym zloc))
        possible-sym)))
```

I think that makes the cascading logic a little clearer.